demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -685,7 +685,9 @@ def validate_no_old_format(self, old_format_files):
         for file_path in old_format_files:
             click.echo(f"Validating old-format file {file_path}")
             yaml_data = get_yaml(file_path)
-            if 'toversion' not in yaml_data:  # we only fail on old format if no toversion (meaning it is latest)
+            # we only fail on old format if no toversion (meaning it is latest)
+            if 'toversion' not in yaml_data and not yaml_data.get('deprecated') or \
+                    ('deprecated' in yaml_data and yaml_data['deprecated'] is False):
```

Comment:
i think you dont need the `or` clause
also need test for the second case

Reply:
I think there are cases when deprecated is in the yml but assigned to False.
Better safe then sorry

Reply:
yea i understand, then the value of `yaml_data.get('deprecated')` will be `False` and it will get in the condition